.github/workflows/pytest.yml

```
@@ -0,0 +1,44 @@
+name: Pytests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # The original 3.6 and 3.7 are no longer in the runners tool cache (too old), so we'll go with our standard
+        python-version: [3.11]
+    steps:
+      - run: sudo apt-get install -y netcdf-bin
+
+      - uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: pip
+
+      - name: Install dependencies
+        run: pip install -r requirements-dev.txt
+
+      - name: Run tests
+        run: pytest
```

```suggestion
        run: python -m pytest
```
That should fix the issue with the tests picking up the local version of the `addmeta` package